src/gennodejs/generate.py

```diff
@@ -728,6 +728,18 @@ def write_constants(s, spec):
         s.write('}')
         s.newline()
 
+def write_type_constants(s, spec):
```

Comment:
all built in types can be constants. see:
https://wiki.ros.org/msg#Constants

i think there is a function here somewhere called is_builtin that has a map of built in types.

Reply:
nvm, I think was really tired when I reviewed this. I see that the else statement will take care of non-string or bool types.